kornia/geometry/plane.py

```diff
@@ -31,7 +33,7 @@
 
 
 def normalized(v: Tensor, eps: float = 1e-6) -> Tensor:
-    return v / batched_dot_product(v, v).add(eps).sqrt()
+    return F.normalize(v, p=2, dim=-1, eps=eps)
```

Comment:
We would avoid using torch functions, since we have also functionalities to port the code to numpy, etc.